package.json

```diff
@@ -43,5 +45,21 @@
   },
   "peerDependencies": {
     "@jupiterone/integration-sdk-core": "^8.41.0"
+  },
+  "auto": {
+    "plugins": [
+      [
+        "npm",
+        {
+          "setRcToken": false,
+          "publishFolder": "."
```

Comment:
I think the publishFolder for this might need to be "./dist" based on the build command above.

Reply:
You might also need to add a `postversion` command above to make sure the package.json file gets copied into the ./dist folder after auto finishes versioning it.  Example: https://github.com/JupiterOne/graph-cisco-umbrella/blob/546be3e1242aa1b99255d428eb4cb9cffb8f9f02/package.json#L32